docs/accessibility.md

```diff
@@ -0,0 +1,89 @@
+Accessibility Guidance
+======================
+
+This document provides guidance and links to useful resources and tools for
+creating web-based content & interfaces which are accessible to the widest
+possible range of users.
+
+It is not intended to be exhaustive but rather to highlight important best
+practices and provide a shared reference which is useful when implementing or
+reviewing UI changes.
+
+## Guidelines
+
+### Mobile
+
+1. **Use media queries to adapt layout to display size** - Use `@media` queries
+   to adjust the layout depending on screen size to provide more usable
+   interfaces on small screens.
```

Comment:
The way I look at it is that content such as Activity Pages has a few major layouts which are appropriate for the typical range of screen sizes of a phone, a tablet and desktop. Those major layouts are likely to be specified by the designer. In the implementation, it is useful to have common variables which define the boundaries of these ranges that are used in the major media queries.

Given that screens at the small end of a range may have a width / height several hundred px different from screens at the large end of a range, it makes sense to adapt/refine the main layout or the layout of individual elements where possible to make the best use of available space. Much of this adaptation can be done just relying on flexbox without any media queries, but there are cases where additional breakpoints are needed. These smaller refinements may be things that we (developers) can do ourselves without needing upfront specification from the person working on the design. An example would be the way that some of the margins disappear in activity pages as the screen size gets smaller towards the lower end of one of the ranges.

Does this answer your question?